README.md

```diff
@@ -64,6 +64,10 @@ This driver supports only one topology key:
 `topology.gke.io/zone`
 that represents availability by zone (e.g. `us-central1-c`, etc.).
 
+### CSI Windows Support
+
+GCE PD driver starts to support CSI Windows with [CSI Proxy] (https://github.com/kubernetes-csi/csi-proxy). It requires to install csi-proxy.exe binary on each Windows node. Please see more details in CSI Windows page (docs/kubernetes/user-guides/windows.md)
```

Comment:
"It requires csi-proxy.exe to be installed on every Windows node"

Reply:
updated, thanks!